sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Generated/MetricsAdvisorClient.cs

```diff
@@ -0,0 +1,3336 @@
+                    var page = LowLevelPageableHelpers.ProcessMessage(_pipeline, message, context, "value", "@nextLink");
+                    nextLink = page.ContinuationToken;
+                    yield return page;
+                } while (!string.IsNullOrEmpty(nextLink));
+            }
+        }
+
+        /// <summary> List feedback on the given metric. </summary>
+        /// <param name="content"> The content to send as the body of the request. </param>
+        /// <param name="skip"> for paging, skipped number. </param>
+        /// <param name="maxpagesize"> the maximum number of items in one page. </param>
+        /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="content"/> is null. </exception>
+        /// <remarks>
+        /// Schema for <c>Request Body</c>:
+        /// <code>{
+        ///   metricId: MetricFeedbackFilterMetricId (required),
+        ///   dimensionFilter: {
+        ///     dimension: Dictionary&lt;string, string&gt; (required)
+        ///   },
+        ///   feedbackType: &quot;Anomaly&quot; | &quot;ChangePoint&quot; | &quot;Period&quot; | &quot;Comment&quot;,
+        ///   startTime: string (ISO 8601 Format),
+        ///   endTime: string (ISO 8601 Format),
+        ///   timeMode: &quot;MetricTimestamp&quot; | &quot;FeedbackCreatedTime&quot;
+        /// }
+        /// </code>
+        /// Schema for <c>Response Body</c>:
+        /// <code>{
+        ///   @nextLink: string,
+        ///   value: [
+        ///     {
+        ///       feedbackType: &quot;Anomaly&quot; | &quot;ChangePoint&quot; | &quot;Period&quot; | &quot;Comment&quot;,
+        ///       feedbackId: MetricFeedbackId,
+        ///       createdTime: string (ISO 8601 Format),
+        ///       userPrincipal: string,
+        ///       metricId: MetricFeedbackMetricId,
+        ///       dimensionFilter: {
+        ///         dimension: Dictionary&lt;string, string&gt;
+        ///       }
+        ///     }
+        ///   ]
+        /// }
+        /// </code>
+        /// Schema for <c>Response Error</c>:
+        /// <code>{
+        ///   message: string,
+        ///   code: string
+        /// }
+        /// </code>
+        /// 
+        /// </remarks>
+        public virtual AsyncPageable<BinaryData> GetMetricFeedbacksAsync(RequestContent content, int? skip = null, int? maxpagesize = null, RequestContext context = null)
+        {
+            Argument.AssertNotNull(content, nameof(content));
+
+            return PageableHelpers.CreateAsyncPageable(CreateEnumerableAsync, ClientDiagnostics, "MetricsAdvisorClient.GetMetricFeedbacks");
+            async IAsyncEnumerable<Page<BinaryData>> CreateEnumerableAsync(string nextLink, int? pageSizeHint, [EnumeratorCancellation] CancellationToken cancellationToken = default)
```

Comment:
@ShivangiReja, @AlexanderSher, what is this code after the return statement?

Reply:
Oh, I see, it's the definition of CreateEnumerableAsync.  Could I be nitpicky and as that we add a carriage return there to make that more clear?

Reply:
Since this is the generated code, could you please create an issue [here](https://github.com/Azure/autorest.csharp/issues/new/choose) with the expected output we want to generate?